spec/support/primerized_flash/expectations.rb

```diff
@@ -0,0 +1,70 @@
+module PrimerizedFlash
+  module Expectations
+    def expect_primerized_error(message)
+      expect_primerized_flash(message:, type: :error)
+    end
+
+    def expect_primerized_flash(message:, type: :success, wait: 20)
+      expected_css = expected_flash_css(type)
+      expect(page).to have_css(expected_css, text: message, wait:)
+    end
+
+    def find_primerized_flash(type:)
+      expected_css = expected_flash_css(type)
+      page.find(expected_css)
+    end
+
+    def expect_and_dismiss_primerized_flash(message: nil, type: :success, wait: 20)
+      expect_primerized_flash(type:, message:, wait:)
+      dismiss_primerized_flash!
+      expect_no_primerized_flash(type:, message:, wait: 0.1)
+    end
+
+    def dismiss_primerized_flash!
+      page.find(".Banner-close button").click # rubocop:disable Capybara/SpecificActions
+    end
+
+    # Clears a toaster if there is one waiting 1 second max, but do not fail if there is none
+    def clear_primerized_flashes
+      if has_css?(".Banner-close button")
+        dismiss_primerized_flash!
+      end
+    end
```

Comment:
I'm not testing the flash aria implementation here, but I'm fine to change it.

Reply:
Warning: it may break things if some other controls are named "Close". Maybe not worth it.

And it's not really about aria implementation, but about adopting the [capybara_accessible_selectors](https://github.com/citizensadvice/capybara_accessible_selectors) approach: "All feature tests should interact with the browser in the same way a screen-reader user would [instead of using css accessors]. This both tests the feature, and ensures the application is accessible."

Reply:
Thanks for trying 👍